stanovy_vpsFree.cz.md

```diff
@@ -154,149 +166,200 @@ informačním systému spolku
 5. Zápis o Schůzi členů musí obsahovat:
   1. název a sídlo Spolku,
   2. místo, datum a dobu konání Schůze členů,
   3. jméno předsedajícího jednání, ověřovatelů a zapisovatele
   4. prezenci přítomných
-  5. popis projednaných bodů programu  6. rozhodnutí Schůze členů s uvedením
-výsledků hlasování
-  7. obsah námitky člena Spolku týkající se rozhodnutí Schůze členů, jestliže
-o to dotyčný žádá
+  5. popis projednaných bodů programu
+  6. rozhodnutí Schůze členů s uvedením výsledků hlasování
+  7. obsah námitky člena Spolku týkající se rozhodnutí Schůze členů, jestliže o
+  to dotyčný žádá
+6. Umožňuje se konání náhradní Schůze členů dle ustanovení § 257 občanského zákoníku.
+
+### §11a Jednání na dálku (per rollam)
+
+1. Jednání Schůze členů dle § 11 může probíhat způsobem umožňujícím dálkovou
+účast; taková schůze může probíhat částečně či zcela ve virtuálním prostoru
+prostřednictvím prostředků komunikace na dálku, a to v informační systému
+(prostředí) Spolku, event. jiným způsobem stanoveným Radou spolku nebo
+příslušným interním řádem
+2. Rada Spolku v pozvánce na Jednání Schůze členů stanoví podmínky dálkové
+účasti na dané Schůzi členů, nebo odkáže na řád pro hlasování Schůze členů na
+dálku (per rollam) přijatý dle § 18 Stanov
+3. Při jednání na dálku mohou účastníci Schůze členů s hlasovacím právem
+hlasovat na dálku; takové hlasování má stejné účinky, jako hlasování v
+přítomnosti
+4. Program Schůze členů, která probíhá částečně či zcela na dálku, se považuje
+za schválený, pokud nedojde k jeho zamítnutí účastníky schůze do 1 hodiny od
+zahájení schůze; bod 5. § 11b o dálkovém hlasování se v tomto případě
+neuplatní.
+
+### §11b Průběh hlasování na dálku (per rollam)
+
+1. Dálkové hlasování je zajištěno vhodnými technickými prostředky
+2. Dálkové hlasování probíhá po jednotlivých návrzích zvlášť
+3. Každý účastník Schůze členů, který hlasuje na dálku, může v každém návrhu
+hlasovat PRO přijetí, PROTI přijetí, nebo se zdržet hlasování; zdržení se
+hlasování musí být aktivním úkonem, který má stejnou formu, jako hlasování PRO
+nebo PROTI
+4. Za přítomného člena podle bodů 1. a 2. §11 se v případě hlasování na dálku
+považuje takový člen, který se zúčastnil hlasování podle předchozího bodu
+5. Je-li o daném návrhu možné hlasovat na dálku, činí termín pro odevzdání
+dálkového hlasu 120 hodin od okamžiku předložení návrhu k hlasování, není-li ve
+schváleném programu schůze u daného návrhu stanoveno jinak
+6. Hlasování na dálku může Rada spolku v mimořádných případech, nebo v
+případech, kdy není praktické svolávat řádnou Schůzi členů, vyvolat i o
+samostatných návrzích mimo řádnou Schůzi členů; v takovém případě se postupuje
+přiměřeně podle § 11, 11a a 11b, a hlasování má stejné účinky, jako by bylo
+provedeno na řádné Schůzi členů
+7. Termín pro odevzdání dálkového hlasu podle předchozího bodu stanoví Rada spolku
+nejméně na 336 hodin; kratší termín může Rada spolku stanovit pouze v
+odůvodněných případech, nejméně však na 24 hodin
```

Comment:
Prijde mi, ze tyhle lhuty jsou prilis kratke, nedovedu si predstavit oduvodneny pripad, kdy bychom mohli chtit provest rozumne uvazenou zmenu stanov, pripadne jinou zasadni zmenu ve fungovani - a mit pritom jasno, jake to bude mit dopady, behem 24 hodin...

Navrhuju termin odevzdani hlasu na 10 pracovnich dni, oduvodnene pripady nejmene 5 pracovnich dni.

Reply:
Souhlas - 24 hodin na to o čem by se mohlo hlasovat je málo. Jen bych to asi nekomplikoval pracovními dny stačí přidat příslušný počet dní kvůli víkendům. 

Počítání v hodinách je asi taky lepší, nebude se muset řešit opičárna typu "X dnů následujících po dni započetí hlasování" apod.

Reply:
prodlouženo na minimálně 120 hodin, zpřísněna podmínka pro využití minimálního času

Reply:
resolved by 6018687

Reply:
jak byste vyřešili ten bod 5.? Nechat tam pevnou lhůtu? Nebo dát radě možnost tu lhůtu jenom prodloužit? Tam by se pak taky musel dát nějakej strop...

Reply:
IMHO pokud bude bezesporné odhlasování programu, tak to může zůstat jak to je (případná šibeniční lhůta do programu neprojde), pokud zůstane krátká lhůta pro zamítnutí programu, tak by to chtělo dát jen možnost prodloužení. Nesmí nastat situace, že by mohly být obě lhůty šibeniční. (a zase a) něco špatného někdo protlačí, b) někdo bude moci napadnout hlasování).

Reply:
souhlas, vyřešil bych to lepším schválením programu - to stejně potřebuje ještě potunit :)